File: java/src/main/java/co/edu/sena/capitulo6/ejercicios/Ejercicio10.java
package co.edu.sena.capitulo6.ejercicios;

import java.util.Scanner;

public class Ejercicio10 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		int i = 0;
		int n = 0;
		System.out.print("Introduce numero : ");
		n = teclado.nextInt();

		while (n > 0) {
			n = n / 10;
			i++;
		}

		System.out.println("tiene " + i + " dígitos conformantes de un numero capicua");
	}

}

File: typescript/src/ejercicios/ejercicio10.ts
(function () {
    let iCantidad = 0;
    let n=0;
     n=parseInt(prompt("Introduce numero : "));
    while (n>0){
      n = n/10;
      iCantidad++;
    }
    console.log("tiene " + iCantidad + " dígitos conformantes de un numero capicua");
})();